setup.py

```diff
@@ -33,48 +31,92 @@ def get_long_description():
 def get_install_requires():
     # type: () -> List[str]
 
-    install_requires = [
-        'alembic', 'cliff', 'colorlog', 'numpy', 'scipy', 'sqlalchemy>=1.1.0', 'tqdm', 'typing']
-    return install_requires
+    return [
+        'alembic',
+        'cliff',
+        'colorlog',
+        'numpy',
+        'scipy',
+        'sqlalchemy>=1.1.0',
+        'tqdm',
+        'typing',
+    ]
 
 
-def get_extras_require():
-    # type: () -> Dict[str, List[str]]
+def get_tests_require():
+    # type: () -> List[str]
 
-    testing_requirements = [
-        'bokeh', 'chainer>=5.0.0', 'cma', 'keras', 'lightgbm', 'mock',
-        'mpi4py', 'mxnet', 'pandas', 'plotly>=4.0.0', 'pytest', 'scikit-optimize',
-        'tensorflow', 'tensorflow-datasets', 'xgboost', 'scikit-learn>=0.19.0',
-        'torch', 'torchvision', 'pytorch-ignite', 'pytorch-lightning',
-    ]
+    return get_extras_require()['testing']
 
-    example_requirements = [
-        'chainer', 'keras', 'catboost', 'lightgbm', 'scikit-learn',
-        'mxnet', 'xgboost', 'torch', 'torchvision', 'pytorch-ignite',
-        'dask-ml', 'dask[dataframe]', 'pytorch-lightning',
 
-        # TODO(Yanase): Update examples to support TensorFlow 2.0.
-        # See https://github.com/optuna/optuna/issues/565 for further details.
-        'tensorflow<2.0.0',
-    ]
+def get_extras_require():
+    # type: () -> Dict[str, List[str]]
 
-    if sys.version_info[:2] > (3, 5,):
```

Comment:
Actually, I once made the same mistake 😄 Please see https://github.com/optuna/optuna/pull/724#discussion_r348909756

Reply:
Oh gosh, let me put this check back..